apps/docs/content/components/VideoPlayer.mdx

```diff
@@ -14,30 +14,37 @@ import {VideoPlayer} from '@primer/react-brand'
 
 ## Examples
 
+<!---
+ Note: These assets will not load in local development. The way gatsby works prefixes the assets URLs which cannot currently be done in these files.
```

Comment:
Technically can work in local dev environment, if you build and serve the docs with a baseURL. Here's a suggestion to explain why we're doing this.

```suggestion
TODO: Fix asset loading in environments that don't use `pathPrefix`. This is a temporary solution to load assets in production deployments, but currently doesn't work in dev mode or preview URLs
```